README.md

```diff
@@ -8,46 +8,49 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/slint-ui/slint)](https://api.reuse.software/info/github.com/slint-ui/slint)
 [![Discussions](https://img.shields.io/github/discussions/slint-ui/slint)](https://github.com/slint-ui/slint/discussions)
 
-Slint is a toolkit to efficiently develop fluid graphical user interfaces for any display: embedded devices and desktop applications. We support multiple programming languages, such as
-Rust, C++, and JavaScript.
+Slint is a toolkit to efficiently design, develop and deploy responsive apps on any screen. Frontends built with Slint can be easily integrated with backends written in different programming languages such as C/C++, Rust, and JavaScript. Slint is developed by SixtyFPS GmbH together with the community. We invite you to use Slint and be part of its community.
 
-Our design goals are:
+## Design Goals
 
+We use the following design goals as a guide when developing Slint
+
+ - **Scalable**: Run on any screen connected to a device, from desktop computers to low end embedded systems.
  - **Lightweight**: Fit into a few hundred kilobytes of RAM and require little processing power.
- - **Straightforward**: Programmers and designers should feel productive and be able to enjoy the design and development process.
-   The APIs should be consistent, easy to use, and intuitive, no matter the target language. High-quality documentation
-   should describe the APIs, teach concepts and how to use them.
- - **Native**: We support many different target platforms, from embedded devices to desktops including mobile and web. Both the user and the developer should feel at
-   home on each platform. The look and feel and experience should match the users' expectations of a
-   native application.
+ - **Intuitive**: Both developers and designers should feel productive and enjoy the design and development process.
+   The APIs should be consistent and easy to use, no matter the target language.
+ - **Native**: Slint apps should match the users' expectations of a native application. Various target platforms such as 
+   embedded devices, desktops, mobile and web should be supported so that both the user and the developer feel comfortable on
+   their platform of choice.
 
-## Quick Start
+## Get Started
```

Comment:
I wonder if we can't just leave out this paragraph.  All the information is somewhere else in the readme

Reply:
Agreed.

Reply:
Removed

Reply:
Eh, I was talking about the Quick Start paragraph,  not the design goals.  @tronical : What paragraph where you talking about?

I think the design goal are good to keep. Or do you think it is not relevant anymore on this page?

Reply:
Yes, I was also talking about the "Quick Start" paragraph. The design goals are good IMO to have on this landing page.

Reply:
removed Quick Start and retained Design Goals